packages/connections/src/components/connection-list/connection-menu.tsx

```diff
@@ -0,0 +1,127 @@
+import { css } from '@emotion/css';
+import React, { useReducer } from 'react';
+import {
+  IconButton,
+  Icon,
+  Menu,
+  MenuItem,
+  Toast,
+  ToastVariant,
+  spacing,
+} from '@mongodb-js/compass-components';
+
+const dropdownButtonStyles = css({
+  color: 'white',
+  position: 'absolute',
+  right: spacing[1],
+  top: spacing[2],
+  bottom: 0,
+});
```

Comment:
Didn't notice before, but it caught my eye that this button is not aligned in the center, probably a bit more top spacing needed here?

<img width="281" alt="image" src="https://user-images.githubusercontent.com/5036933/140108925-c0556a7e-8b99-4347-92f8-92001b863bbd.png">

Reply:
Nice catch, updated.
<img width="290" alt="Screen Shot 2021-11-03 at 1 15 46 PM" src="https://user-images.githubusercontent.com/1791149/140139800-a8230749-586e-4e79-ae69-54ed8a1b078c.png">